### ISSUE TITLE
MobileNetV1: Adding 

### ISSUE THREAD

**Issue by fzyzcjy** [keras-team/keras-applications#117]

**Comment by taehoonlee** [comment]
@fzyzcjy, Yes, I agree with you. The l2 is essential for training. But it is not easy to add them.
1. I think that adding l2 into only a single model is not good. In order to add them, we'd better apply them into all the models in the keras-applications. We must survey all the original papers or repositories.
2. In the current APIs, we must change all the lines of individual layer definitions. It is not an efficient job. We'd better propose a new API first, like `arg_scope` in TensorFlow.

**Comment by fzyzcjy** [comment]
@taehoonlee I agree with you. So shall we create an issue in tf repo?
